README.md

```diff
@@ -1,6 +1,10 @@
 # OMR Checker
 
-Read OMRs fast and accurately using a scanner 🖨 or your phone 🤳.
+Read Optical Mark Recognitions (OMRs) fast and accurately using a scanner 🖨 or your phone 🤳.
```

Comment:
If we're adding an FAQ just below the intro, then we may not need to repeat Optical Mark Recognitions.
```suggestion
Read OMR sheets fast and accurately using a scanner 🖨 or your phone 🤳.
```